### PR TITLE
[MIG] barcodes_generator_product: Migration to 14.0

### PR DESCRIPTION
**Pull request by hugho-ad**

### CODE REVIEW

**Comment by hugho-ad** [comment]
@hparfr

could you review pls

**Comment by hugho-ad** [comment]
@Zar21 could you review pls

File: barcodes_generator_product/models/product_template.py [hunk 1]
```diff
@@ -27,7 +25,6 @@ class ProductTemplate(models.Model):
 
     generate_type = fields.Selection(
         string="Generate Type",
-        selection=_GENERATE_TYPE,
```
Comment (legalsylvain) [inline barcodes_generator_product/models/product_template.py]: Why did you removed this line ?

**Review by legalsylvain — APPROVED** [review]
One question. Otherwise. Lgtm. Thanks for porting this module.
Note : you can add me as maintainer in the manifest.

File: barcodes_generator_product/models/product_template.py (hunk 1 above)
Comment (hugho-ad) [reply barcodes_generator_product/models/product_template.py]: because it's a related field
does not make sense re-defined selection attribute

Comment (hugho-ad) [reply barcodes_generator_product/models/product_template.py]: see https://github.com/odoo/odoo/blob/14.0/odoo/fields.py#L2250-L2251

**Review by moylop260 — APPROVED** [review]
👍

**Comment by hugho-ad** [comment]
@misern2 could you review pls

**Review by misern2 — APPROVED** [review]
:+1:

File: barcodes_generator_product/models/product_template.py (hunk 1 above)
Comment (legalsylvain) [reply barcodes_generator_product/models/product_template.py]: Thanks a lot ! I didn't know.

**Comment by hugho-ad** [comment]
@pedrobaeza 

could you merge?

**Pull request merged**